stdlib/ballerina-grpc/src/main/java/org/ballerinalang/net/grpc/nativeimpl/connection/server/Error.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.ballerinalang.net.grpc.nativeimpl.connection.server;
+
+import io.grpc.Status;
+import io.grpc.StatusRuntimeException;
+import io.grpc.stub.StreamObserver;
+import org.ballerinalang.bre.Context;
+import org.ballerinalang.model.types.TypeKind;
+import org.ballerinalang.model.values.BStruct;
+import org.ballerinalang.model.values.BValue;
+import org.ballerinalang.natives.AbstractNativeFunction;
+import org.ballerinalang.natives.annotations.Argument;
+import org.ballerinalang.natives.annotations.BallerinaFunction;
+import org.ballerinalang.natives.annotations.Receiver;
+import org.ballerinalang.natives.annotations.ReturnType;
+import org.ballerinalang.net.grpc.MessageConstants;
+import org.ballerinalang.net.grpc.MessageUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Native function to send server error the caller.
+ *
+ * @since 0.96.1
+ */
+@BallerinaFunction(
+        packageName = "ballerina.net.grpc",
+        functionName = "error",
+        receiver = @Receiver(type = TypeKind.STRUCT, structType = MessageConstants.SERVER_CONNECTION,
+                structPackage = MessageConstants.PROTOCOL_PACKAGE_GRPC),
+        args = {@Argument(name = "serverError", type = TypeKind.STRUCT, structType = "GrpcServerError",
+                structPackage = MessageConstants.PROTOCOL_PACKAGE_GRPC)},
+        returnType = @ReturnType(type = TypeKind.STRUCT, structType = "ConnectorError",
+                structPackage = MessageConstants.PROTOCOL_PACKAGE_GRPC),
+        isPublic = true
+)
+public class Error extends AbstractNativeFunction {
```

Comment:
Error being a noun doesn't sound as call to a native function. Is this something we agreed when we design API?